charts/buildbuddy-enterprise/requirements.lock

```diff
@@ -26,5 +26,8 @@ dependencies:
 - name: grafana
   repository: https://grafana.github.io/helm-charts
   version: 6.6.2
-digest: sha256:59082060763f0c9fe572f51c1e20b50a1c202b4819d51a43d9861bf1ddbe965c
-generated: "2024-06-14T21:09:22.039382286Z"
+- name: clickhouse
+  repository: https://charts.bitnami.com/bitnami
+  version: 6.2.0
```

Comment:
curious: what version of ClickHouse it is?

Reply:
The chart 6.2.0 comes with this image
```
image:
  registry: docker.io
  repository: bitnami/clickhouse
  tag: 24.4.1-debian-12-r4
```

https://artifacthub.io/packages/helm/bitnami/clickhouse/6.2.0?modal=values&path=image.registry

The image is built with this https://github.com/bitnami/containers/tree/main/bitnami/clickhouse/24/debian-12

Reply:
Shall we keep it at 23.8? We are using 23.8. in production. https://github.com/buildbuddy-io/buildbuddy-internal/blob/6c6f41115594833c36730c4d6fcf068ac5eaba42/enterprise/deployment/prod/us-west1/clickhouse.prod.yaml#L41

Reply:
Will do.

Let me test it locally a bit first to verify. Will update the PR after. 🙏

Reply:
Pinned the image to 23.8 in `values.yaml`